### DISCUSSION TITLE
[Intellisense] Enabling intellisense in a lisp file

### DISCUSSION THREAD

**Discussion by embalmedhead** [tailwindlabs/tailwindcss#13010] [Help]
I'm using a lisp ([Racket](https://racket-lang.org/)) to write my HTML. Here's a simple example of a component definition in a Racket file (extension `.rkt`):

```racket
(define (example . text)
  `(p [(class "font-weight-bold color-hotpink")] ,@text))
```

I'd like to enable intellisense here.  I've tried adding regexes to `tailwindCSS.experimental.classRegex` and enabling the racket language by adding `"plaintext": "racket"` to `tailwindCSS.includeLanguages`, but nothing seems to make any dent. I'm using VS Code.

Does anyone have any ideas? Thank you in advance.

**Comment by wongjn** [comment]
You'd want the `tailwindCSS.includeLanguages` to be reversed, such that the key is the "new" language ID, and the value is any one of the extension's built-in languages. Otherwise, consider checking your regexes match properly.
